packages/signers/package.json

```diff
@@ -94,6 +99,10 @@
     "@usecapsule/web-sdk": "^0.29.4",
     "@web3auth/base": "^7.1.0",
     "@web3auth/modal": "^7.1.1",
+    "@0xpass/passport": "^1.0.0",
+    "@0xpass/passport-viem-alchemy": "0.0.1",
```

Comment:
what's this sdk? any reason not to have it in your signer impl in aa-signers?

Reply:
`@0xpass/passport` is our core typescript sdk for interacting with our network, and `@0xpass/passport-viem-alchemy` gives us access to create a custom viem wallet client which we need for some of our functionality, we made an alchemy specific one, so that we can keep it in sync with whatever `viem` version the `aa-signers` package is using